pkgs/tools/misc/mongodb-tools/goconvey.nix

```diff
@@ -0,0 +1,18 @@
+{ stdenv, goPackages, fetchFromGitHub, callPackage }:
```

Comment:
I added v1.5.0 to go-packages, but mongo-tools lists a specific revision, which is what I use here.